hal/src/sercom/v2/spi_future.rs

```diff
@@ -204,6 +204,7 @@ type Data = u32;
 //=============================================================================
 
 /// Trait used to verify the [`SpiFuture`] buffer length
+#[allow(clippy::len_without_is_empty)]
```

Comment:
I'm due to overhaul `SpiFuture` after using it a good amount. A lot will change.